docs/v3/develop/task-runners.mdx

```diff
@@ -29,40 +31,20 @@ such as for operations mapped across a dataset.
 
 ## Configure a task runner
 
-To configure your flow to use a specific task runner, provide the runner to the `task_runner` keyword of the flow decorator.
-
-To submit work to the runner, use the task's `.submit()` method.
-This method returns a [`PrefectFuture`](https://reference.prefect.io/prefect/futures/#prefect.futures.PrefectFuture), which is a Prefect object that contains:
-- a reference to the payload returned by the task;
-- and a [`State`](https://reference.prefect.io/prefect/server/schemas/states/), which is a Prefect object indicating the state of the task run.
```

Comment:
this section was repeated down below, and the useful warning was moved down to the introduction of `PrefectFuture`